test/e2e/autoscale.go

```diff
@@ -262,7 +270,7 @@ func assertScaleDown(ctx *TestContext) {
 	}
 
 	// Account for the case where scaling up uses all available pods.
-	ctx.t.Log("Wait for all pods to terminate.")
+	ctx.t.Logf("Wait for all pods to terminate.")
```

Comment:
`ctx.t.Logf` should have been a `ctx.logf` right?
```suggestion
	ctx.logf("Wait for all pods to terminate.")
```